prediction_market_agent/agents/microchain_agent/memory.py

```diff
@@ -0,0 +1,22 @@
+# inspired by crewAI's LongTermMemory (https://github.com/joaomdmoura/crewAI/blob/main/src/crewai/memory/long_term/long_term_memory.py)
+from typing import Any, Dict, Sequence
+
+from prediction_market_agent.db.db_storage import DBStorage
+from prediction_market_agent.db.models import LongTermMemories
+
+
+# In the future, create a base class which this class extends.
+class LongTermMemory:
+    def __init__(self, task_description: str, storage: DBStorage):
```

Comment:
The only time this is called in this PR, it's called with `storage=DBStorage()`. If that's the only way we expect it to be called, we could just remove the arg, and initialize that here with `self.storage = DBStorage()`

Reply:
It is the only way I can imagine at this time, so implementing your suggestion